modules/core/src/test/java/org/apache/ignite/spi/discovery/tcp/TcpDiscoverySelfTest.java

```diff
@@ -1976,7 +1976,8 @@ private void checkFailedCoordinatorNode(SegmentationPolicy segPlc) throws Except
 
             ignite1.configuration().getDiscoverySpi().failNode(coordId, null);
 
-            assertTrue(failedLatch.await(2000, MILLISECONDS));
+            // Wait for the configured timeout + other possible code delays.
+            assertTrue(failedLatch.await(ignite1.configuration().getFailureDetectionTimeout() + 50, MILLISECONDS));
```

Comment:
magic number?

Reply:
Same here. 

There is a comment: "Wait for the configured timeout + other possible code delays."

We always can be hit by GC pause, timer granulation, delays in other code. Not giving additional time to wait can lead to flaky tests. Makes sense?